README.md

```diff
@@ -39,3 +39,37 @@ This will create a container called `fedora-toolbox-<version-id>`.
 [user@hostname ~]$ toolbox enter
 ⬢[user@toolbox ~]$
 ```
+
+## Install dependencies, build and install toolbox
+
+### Install dependencies:
+The following dependencies is required to build and install:
+- meson
+- go-md2man
+- shellcheck
+- bash-completion
+
+### Build and install `toolbox`:
+
+> **Warning:** These steps must be done in source directory
+
+1. Create build directory:
+```
+mkdir build
+```
+2. Enter in build directory:
+```
+cd build
+```
+3. Configure `toolbox` using `meson`:
+```
+meson ..
```

Comment:
I wonder if we should combine the first three steps into one as:
```
$ meson builddir
```

No strong opinion.

However, I think that we should at least mention the `-Dprofile_dir` option. Currenty, most people will need `-Dprofile_dir=/etc/profile.d`.

(The `/etc/profile.d` situation is a bit odd.

Modern distributions try to have clear separation between configuration shipped by the OS vendor and changes made by the local system administrator - somewhere inside `/usr` for the former and `/etc` for the latter. eg., /usr/lib/os-release, /usr/lib/sysctl.d, etc.. However, in most cases, this hasn't happened with the shell / environment variable configuration setup. Most distributions continue to dump everything in `/etc/profile.d`.

Flatpak also had to deal with this: https://github.com/flatpak/flatpak/commit/3b64fbc4f51692ca821c1e197b671ac4b22bf726

We currently default to `/usr/share/profile.d` to be future proof and apply some pressure on distributions, but for distributions still using the old pattern (ie., most of them) we have to use the build flag.)